travis.sh

```diff
@@ -93,7 +93,11 @@ case "$UPSTREAM_WORKSPACE" in
         ;;
     http://* | https://*) # When UPSTREAM_WORKSPACE is an http url, use it directly
         travis_run wstool init .
-        travis_run wstool merge $UPSTREAM_WORKSPACE
+        # Handle multiple rosintall entries.
+        IFS=','  # Multiple URLs can be given separated by comma.
+        for rosinstall in $UPSTREAM_WORKSPACE; do
+            travis_run wstool merge $rosinstall
```

Comment:
supply the appropriate command line argument such that the easier rosinstall files listed are *overwritten* by later rosinstall files. currently travis is prompting the user and failing